dune

```diff
@@ -10,7 +10,7 @@
  ; don't stop building because of warnings
  (dev
   (flags
-   (:standard -warn-error -a)))
+   (:standard -warn-error -a -w -67)))
```

Comment:
Why did you exclude `-67` warnings?

Reply:
it's about unused functor parameters, so unlikely to be problematic, and most annoyingly is triggered on `.mli` files, forcing you to write `module Foo (_: TARG): ...` instead of `module Foo (MeaningfulArgName: TARG): ...`. Which I find less helpful.

Reply:
OK fine :)